src/pyTRLCConverter/render_config.py

```diff
@@ -0,0 +1,197 @@
+            pass
+
+        return status
+
+    def _is_package_match(self, item: dict, trlc_package: str) -> bool:
+        """Checks if the given TRLC package matches the package pattern in the given item.
+
+        Args:
+            item (dict): Single render configuration item.
+            trlc_package (str): The TRLC package.
+
+        Returns:
+            bool: If the given TRLC package matches the package pattern in the given item.
+        """
+        it_matches = False
+
+        if "package" in item:
+            if re.match(item["package"], trlc_package):
+                it_matches = True
+
+        return it_matches
+
+    def _is_type_match(self, item: dict, trlc_type: str) -> bool:
+        """Checks if the given TRLC type matches the type pattern in the given item.
+
+        Args:
+            item (dict): Single render configuration item.
+            trlc_type (str): The TRLC type.
+
+        Returns:
+            bool: If the given TRLC type matches the type pattern in the given item.
+        """
+        it_matches = False
+
+        if "type" in item:
+            if re.match(item["type"], trlc_type):
+                it_matches = True
+
+        return it_matches
+
+    def _is_type_attribute_match(self, item: dict, trlc_type_attribute: str) -> bool:
+        """Checks if the given TRLC type attribute matches the type attribute pattern in the given item.
+
+        Args:
+            item (dict): Single render configuration item.
+            trlc_type (str): The TRLC type attribute.
+
+        Returns:
+            bool: If the given TRLC type attribute matches the type attribute pattern in the given item.
+        """
+        it_matches = False
+
+        if "attribute" in item:
+            if re.match(item["attribute"], trlc_type_attribute):
+                it_matches = True
+
+        return it_matches
+
+    def get_format_specifier(self, trlc_package: str, trlc_type: str, trlc_type_attribute: str) -> str:
+        """Returns the format specifier for the given TRLC package, type and attribute.
+
+        Args:
+            trlc_package (str): The TRLC package.
+            trlc_type (str): The TRLC type.
+            trlc_type_attribute (str): The TRLC type attribute.
+
+        Returns:
+            str: The format specifier for the given TRLC package, type and attribute.
+        """
+        format_specifier = RenderConfig.FORMAT_SPECIFIER_PLAIN
+
+        if "renderCfg" in self._cfg:
+            for item in self._cfg["renderCfg"]:
+                match_list = []
+
+                match_list.append(self._is_package_match(item, trlc_package))
+                match_list.append(self._is_type_match(item, trlc_type))
+                match_list.append(self._is_type_attribute_match(item, trlc_type_attribute))
+
+                # All of the available ones must match!
+                if all(match_list):
+                    if "format" in item:
+                        format_specifier = item["format"]
```

Comment:
regarding my earlier comment on package ordering. Here I miss a break/return out of the for loop to make the 1st hit win, not the last.